lib/countly.js

```diff
@@ -491,8 +504,11 @@
                 //empty event queue
                 if (eventQueue.length > 0) {
                     toRequestQueue({ events: JSON.stringify(eventQueue) });
-                    eventQueue = [];
-                    store("cly_event", eventQueue);
+                    // if test mode enabled; don't make empty eventQueue
+                    if (!Countly.test_mode) {
+                        eventQueue = [];
+                        store("cly_event", eventQueue);
+                    }
```

Comment:
Don't think we need this too, as I dont think you test this method. Again just to make this lib as small as possible